Automattic/wp-calypso
# Add "recycle content" to the Basic plan

#### Proposed Changes

This small change adds the "Recycle Content" feature to the Basic plan in the pricing page.

#### Testing Instructions

* Start JP Cloud locally with this change. Basic plan should include Recycle content:

<img width="1089" alt="CleanShot 2023-01-31 at 10 45 25@2x" src="https://user-images.githubusercontent.com/36671565/215725436-50126597-0531-4bff-8a2c-ecf794db4dc0.png">


Related to 1203640801049718-as-1203800356216183
